src/vcf/store_parse_policy.cpp

```diff
@@ -112,6 +115,28 @@ namespace ebi
     void StoreParsePolicy::handle_header_line(ParsingState & state)
     {
         state.set_samples(m_grouped_tokens);
+        check_each_sample_appears_only_once(state);
+    }
+    
+    void StoreParsePolicy::check_each_sample_appears_only_once(ParsingState const &state) const
+    {
+        using util::operator<<;
+        std::set<std::string> unique_samples;
+        std::vector<std::string> repeated_sample_names;
```

Comment:
When a sample is a triplicate it is reported twice, which is somewhat confusing. I think this would better be a set instead of a vector.

Reply:
I did that on purpose, suppose you have a big list (thousands of samples), and it says 1 duplicate. you remove it, and after validating, it reports the same duplicate again. I think that would be way more confusing than having the reported list with repeated names.

Reply:
message changed to 
> "The header line has 3 repeated sample names. Please remove these names from the header line and the data lines: HG00096 (2 times), HG00097."